go.mod

```diff
@@ -48,11 +50,13 @@ require (
 	github.com/vmihailenco/tagparser v0.1.2 // indirect
 	github.com/zclconf/go-cty v1.12.1 // indirect
 	golang.org/x/crypto v0.0.0-20220517005047-85d78b3ac167 // indirect
-	golang.org/x/net v0.0.0-20211208012354-db4efeb81f4b // indirect
-	golang.org/x/sys v0.0.0-20220503163025-988cb79eb6c6 // indirect
-	golang.org/x/text v0.3.7 // indirect
+	golang.org/x/net v0.0.0-20220722155237-a158d28d115b // indirect
+	golang.org/x/sys v0.0.0-20220722155257-8c9f86f7a55f // indirect
+	golang.org/x/text v0.4.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
 	google.golang.org/genproto v0.0.0-20220608133413-ed9918b62aac // indirect
-	google.golang.org/grpc v1.50.1 // indirect
+	google.golang.org/grpc v1.51.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
+
+replace github.com/hashicorp/terraform-plugin-mux v0.7.0 => github.com/k-yomo/terraform-plugin-mux v0.0.0-20221218141521-7a01a5a64e24
```

Comment:
Just flagging it so it doesn't get merged like this. Moving the maxitems validation to code in the short term is ok if getting your PR merged into `terraform-plugin-mux` turns out to be too slow.

Reply:
The below update contains the PR for fix!
https://github.com/elastic/terraform-provider-elasticstack/pull/235